CHANGELOG.md

```diff
@@ -2,30 +2,37 @@
 
 _released `2023-03-21`_
 
-Version `14.0.0` is a breaking change release that contains some backwards-incompatible changes that may require you to provide additional meta information about your app (if that information isn't already provided). If you are missing this information, you will be blocked from making major updates to your application (`zapier promote`, `zapier upload`, `zapier push`).
+Version `14.0.0` is a breaking change release that contains several important upgrades and deprecations. Here is a brief breakdown of the most notable changes (see further below for a more detailed list of changes):
+
+1. `alters_custom_fields` is no longer set to `true` all the time. Developers should now set the flag appropriately. This will signal to the editor when dynamic dropdown choices have to be re-fetched (i.e. user picked another spreadsheet so we need to refresh the dropdowns that act on column/rows).
+1. Starting on version `14.0.0` CLI integrations will need to define `can_paginate` for every operation that supports pagination. We will preserve the behavior for apps built on platform version `13.0.0` or lower.
```

Comment:
````suggestion
2. [`bundle.meta.page`](https://github.com/zapier/zapier-platform/blob/main/packages/cli/README.md#bundlemeta) no longer defaults to 0. Starting on v14, it will be only available if [`canPaginate`](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@14.0.0/packages/schema/docs/build/schema.md#basicpollingoperationschema) is true. Set `canPaginate` accordingly if your function expects to see `bundle.meta.page`:

   ```
   triggers: {
     contact: {
       operation: {
         canPaginate: true  // <- set this appropriately on v14
       }
     }
   }
   ```
````

While writing this, I figured we probably should update the doc about [`bundle.meta`](https://github.com/zapier/zapier-platform/blob/d5f384165a83e02341c1bb74f615d52b6ac11a67/packages/cli/README.md#bundlemeta) and mention `page` doesn't always exist.